docs/source/configuration/authorization.mdx

```diff
@@ -408,6 +416,192 @@ The response retains the initial request's shape but returns `null` for unauthor
+
+#### Usage
+
+To use the `@policy` directive in a subgraph, you can [import it from the `@link` directive](/federation/federated-types/federated-directives/#importing-directives) like so:
+
+```graphql
+extend schema
+  @link(
+    url: "https://specs.apollo.dev/federation/v2.5",
+    import: [..., "@policy"])
+```
+
+The `@policy` directive is defined as follows:
+
+```graphql
+scalar federation__Policy
+directive @policy(policies: [[federation__Policy!]!]!) on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
+```
+
+Using the `@policy` directive requires a [Supergraph plugin](../customizations/overview) to evaluate the authorization policies. You can do this with a [Rhai script](../customizations/rhai/) or [coprocessor](../customizations/coprocessor). Refer to the following [Rhai script](#usage-with-a-rhai-script) and [coprocessor](#usage-with-a-coprocessor) examples for more information. (Although a [native plugin](../customizations/native) can also evaluate authorization policies, we don't recommend using it.)
+
+#### Combining policies with `AND`/`OR` logic
+
+Authorization validation uses **AND** logic between the elements in the inner-level `policies` array, where a request must include _all_ elements in the inner-level `policies` array to resolve the associated field or type. For the following example, a request would need `policy1` **AND** `policy2` **AND** `policy3` to be authorized:
+
+```graphql
+@policy(policies: [["policy1", "policy2", "policy3"]])
+```
+
+Alternatively, to introduce **OR** logic you can use nested arrays. For the following example, a request would need `policy1` **OR** `policy2` **OR** `policy3` to be authorized:
+
+```graphql
+@policy(policies: [["policy1"], ["policy2"], ["policy3"]])
+```
+
+You can nest arrays and elements as needed to achieve your desired logic. For the following example, its syntax requires requests to have either (`policy1` **AND** `policy2`) **OR** just `policy3` to be authorized:
+
+```graphql
+@policy(policies: [["policy", "policy2"], ["policy3"]])
+```
+
+##### Usage with a Rhai script
+
+The `policies` argument contains a list of strings with no formatting constraints, meaning you can use them to store Rhai code.
+
+<ExpansionPanel title="Click to expand">
+
+As an example, the following schema defines policies as boolean expressions that can be evaluated in Rhai:
+
+```graphql
+type Query {
+  me: User @join__field(graph: ACCOUNTS) @policy(policies: [["claims[`kind`] == `user`"]])
+}
+
+type User
+  @join__owner(graph: ACCOUNTS)
+  @join__type(graph: ACCOUNTS, key: "id") {
+  id: ID! @join__field(graph: ACCOUNTS)
+  username: String @join__field(graph: ACCOUNTS)
+  username: String @join__field(graph: ACCOUNTS) @policy(policies: [["claims[`roles`].contains(`support`)"]])
+}
```

Comment:
We should be showing subgraphs in code snippets, not supergraph SDL. Most users should never look at or worry about this, so it's unnecessary complexity.

This looks like a single subgraph, so let's just show the code for that subgraph (also TODO, adjust this to not look like eval):

```suggestion
type Query {
  me: User @policy(policies: [["claims[`kind`] == `user`"]])
}

type User {
  id: ID!
  username: String
  username: String @policy(policies: [["claims[`roles`].contains(`support`)"]])
}
```